code/turf/turf_flock.dm

```diff
@@ -270,6 +270,7 @@ turf/simulated/floor/feather/proc/bfs(turf/start)//breadth first search, made by
 	..()
 	setMaterial(getMaterial("gnesis"))
 	src.health = src.max_health
+	src.AddComponent(/datum/component/flock_protection, FALSE, TRUE, TRUE)
```

Comment:
Are you sure we want to be notified when someone attacks a converted wall? That seems like it would get irritating real quick.

Reply:
It will only be done for new attackers.